# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1376876999 14400
#      Sun Aug 18 21:49:59 2013 -0400
# Node ID f052d7ccb7dc99c35cc423b0c3fd322a8aa46662
# Parent  8201f78a006458c4c063c1d4b1b40eda55724f0c
Allow empty TeX list {}

* libinterp/corefcn/oct-tex-parser.yy (scoped_element_list): Allow
empty list, as {}.

diff --git a/libinterp/corefcn/oct-tex-parser.yy b/libinterp/corefcn/oct-tex-parser.yy
--- a/libinterp/corefcn/oct-tex-parser.yy
+++ b/libinterp/corefcn/oct-tex-parser.yy
@@ -177,16 +177,18 @@ combined_script_element		: subscript_ele
 string_element_list		: string_element
 				  { $$ = new text_element_list ($1); }
 				| string_element_list string_element
 				  { $1->push_back ($2); $$ = $1; }
 				;
 
 scoped_string_element_list	: START string_element_list END
 				  { $$ = $2; }
+				| START END
+				  { $$ = new text_element_list (); }
 				;
 
 string				: /* empty */
 				  { parser.set_parse_result (new text_element_string ("")); }
 				| string_element_list
 				  { parser.set_parse_result ($1); }
 				;
 
